src/ZohoDatabaseModelSync.php

```diff
@@ -136,7 +136,7 @@ public function synchronizeDbModel(AbstractZohoDao $dao, $twoWaysSync, $skipCrea
                 case 'Multiselect Pick List':
                     $type = 'string';
                     // $field['maxlength'] is not enough
-                    $length = 200;
+                    $length = 255;
```

Comment:
This is a poor fix IMO.
The 255 length can be reached by selecting a lot of items (for instance a pick list with all possible countries selected).

We should instead change the type to `text` and remove the length.

See: http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/types.html#text